components/ArticleCard/ArticleCardContent.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/image';
+
```

Comment:
```suggestion
import classNames from 'classnames';
```

classNames를 사용하기 위해서는 import 해주세요!